### ISSUE TITLE
pypi sdists are missing tests

### ISSUE THREAD

**Issue by dvzrv** [geopy/geopy#385]
Hi! I'm packaging geopy for Arch Linux.
When trying to package 1.20.0 I realized, that the tests are missing from pypi's sdist source tarball.
We usually use pypi to track and build source tarballs.

I guess the MANIFEST.in doesn't state this correctly. It should be `recursive-include test/* *`.

It would be awesome, if the next source tarball on pypi would include the tests. Thanks!

**Comment by KostyaEsmukov** [comment]
Fair point, I believe I wanted to do this at some point but I guess I haven't yet.

I'll address this in the next release. Thanks for reminding!

**Comment by KostyaEsmukov** [comment]
Fixed in 0f0ac3e, will be released as 1.21.0 soon

**Comment by KostyaEsmukov** [comment]
Also 524bc03 and b81969c.

Tests from sdist seem to work fine:

```
docker run -it --rm -v `pwd`/dist:/d python:3.6 bash
$ cd
$ tar xaf /d/geopy-1.21.0.tar.gz
$ cd geopy-1.21.0/
$ pip install .[dev]
$ pytest
<...>
collected 513 items
<...>
====== 305 passed, 208 skipped in 67.45s (0:01:07) =====
<...>
```

Let me know if you need anything else.

**Comment by KostyaEsmukov** [comment]
I have just released 1.21 which contains tests in sdist: https://pypi.org/project/geopy/1.21.0/
